pkg/containers/runtime/sockets.go

```diff
@@ -0,0 +1,34 @@
+package runtime
+
+import (
+	"fmt"
+	"os"
+)
+
+type Sockets struct {
+	sockets map[RuntimeId]string
```

Comment:
I am wondering if there will be any scenario where sockets are registered on runtime...

Reply:
Maybe if we detect the socket has been moved? If we wouldn't allow runtime registering, what would you suggest the API be instead?